diff --git a/node_modules/@maplibre/maplibre-react-native/android/build.gradle b/node_modules/@maplibre/maplibre-react-native/android/build.gradle
index 3eb8289..71ae05f 100644
--- a/node_modules/@maplibre/maplibre-react-native/android/build.gradle
+++ b/node_modules/@maplibre/maplibre-react-native/android/build.gradle
@@ -1,21 +1,3 @@
-buildscript {
-    ext.getExtOrDefault = { name ->
-        return rootProject.ext.has(name) ? rootProject.ext.get(name) : project.properties['org.maplibre.reactnative.' + name]
-    }
-
-    repositories {
-        google()
-        mavenCentral()
-    }
-
-    dependencies {
-        classpath "com.android.tools.build:gradle:8.7.2"
-        // noinspection DifferentKotlinGradleVersion
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${getExtOrDefault('kotlinVersion')}"
-    }
-}
-
-
 apply plugin: "com.android.library"
 apply plugin: "kotlin-android"
 
@@ -29,6 +11,10 @@ def getConfigurableExtOrDefault(name) {
     return rootProject.ext.has("org.maplibre.reactnative." + name) ? rootProject.ext.get("org.maplibre.reactnative." + name) : project.properties["org.maplibre.reactnative." + name]
 }
 
+def getExtOrDefault(name) {
+    return rootProject.ext.has(name) ? rootProject.ext.get(name) : project.properties['org.maplibre.reactnative.' + name]
+}
+
 android {
     namespace "org.maplibre.reactnative"
 
